flow360/component/simulation/meshing_param/params.py

```diff
@@ -70,9 +72,32 @@ class MeshingParams(Flow360BaseModel):
         + "However the impact on regions without close proximity is negligible.",
     )
 
-    surface_layer_growth_rate: float = pd.Field(
+    ##------------ Default settings ------------##
+
+    ##::   Default surface edge settings
```

Comment:
Can we add split just like models here?

Reply:
<img width="235" alt="image" src="https://github.com/user-attachments/assets/01f7edbc-8bae-465a-bb12-26c87e16915c">